plugins/readtimer.koplugin/main.lua

```diff
@@ -12,6 +12,8 @@ local time = require("ui/time")
 local _ = require("gettext")
 local T = require("ffi/util").template
 
+local restore_scheduled_time, restore_last_interval_time -- to keep scheduled timer across views
```

Comment:
I'd even move them inside `ReadTimer` (and access them through `ReadTimer.`), instead of using module-locales ;).

(And possibly explicitly zero-init the two *instance*  variables currently in there, `time` & `last_interval_time` in `init` to make the distinction clearer).

A comment above the declaration of the static ones àla AutoSTandby would also be good ;).

Reply:
> I'd even move them inside `ReadTimer` (and access them through `ReadTimer.`), instead of using module-locales ;).

What's the difference?

Reply:
I mean, the benefits?
Should it be so everywhere, for example
https://github.com/koreader/koreader/blob/deacf70560760eb18ca921d741d3210cf09f7c07/plugins/coverbrowser.koplugin/main.lua#L55-L56

Reply:
> What's the difference?

Not going through an extra upvalue (I *think*).

> Should it be so everywhere

Not for the sake of it, not (i.e., if someone were to refactor something in the vicinity, eeeeh, why not; but otherwise, let sleeping dogs lie ;)).

Reply:
There's also a cosmetic/stylistic aspect to it; I find module-locales more error-prone ;).